### DISCUSSION TITLE
Why not support `text-md` for Font Size ?

### DISCUSSION THREAD

**Discussion by wzc520pyfm** [tailwindlabs/tailwindcss#13804] [Ideas]
![image](https://github.com/tailwindlabs/tailwindcss/assets/69044080/d3ba220f-6621-4e1d-93e0-83047001a7a9)

Maybe add `text-md` as an alias for `text-base` ?

**Comment by donaald795** [comment]
Your article content is being very much interested, I am very impressed with your post. [My Indigo Card](https://www-indigocard.com/)

**Comment by GorlikItsMe** [comment]
I just lost half hour searching why `text-md` dont work.

Fix:
```css
.text-md {
  @apply text-base
}
```

**Comment by matagaralph** [comment]
> ![image](https://private-user-images.githubusercontent.com/69044080/337293500-d3ba220f-6621-4e1d-93e0-83047001a7a9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIyMzE5ODksIm5iZiI6MTcyMjIzMTY4OSwicGF0aCI6Ii82OTA0NDA4MC8zMzcyOTM1MDAtZDNiYTIyMGYtNjYyMS00ZTFkLTkzZTAtODMwNDcwMDFhN2E5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI5VDA1NDEyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIzOGUyYjBhNTA1YmY4NTQ4Njc2Y2Y5ZWU4MWY5MTIyOTc4NWM1NTAxZjI4YzRiNmRkMGRjNGE4NDk4NzYyM2UmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ZU9lC4q4DUjEIjZt0wIGHl3vLcCEY6Lncf65AICNflw)
> 
> Maybe add `text-md` as an alias for `text-base` ?

It's because i think the creators found text-base to better than text-md. I understand and feel you but text-base sounds logical.

**Comment by cupid20103** [comment]
Hmm~ Funny! Your post looks impressive. I am also want to know about that 😁
